WordPress/src/main/java/org/wordpress/android/ui/posts/PromoDialog.kt

```diff
@@ -56,7 +56,7 @@ class PromoDialog : AppCompatDialogFragment() {
         title: String,
         message: String,
         positiveButtonLabel: String,
-        @DrawableRes drawableResId: Int,
+        @DrawableRes drawableResId: Int = -1,
```

Comment:
nitpick: can we define the -1 as a  `const UNDEFINED_RES_ID`  and use it for assignment here, and also in the `if (drawableResId == -1)` a few lines below?

Reply:
Thanks for your review, this was not nitpick at all, I've address it together with other comments.

Should be ready for another look!